simpleclient_log4j/src/main/java/io/prometheus/client/log4j/InstrumentedAppender.java

```diff
@@ -0,0 +1,93 @@
+package io.prometheus.client.log4j;
+
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Counter;
+import org.apache.log4j.Appender;
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Level;
+import org.apache.log4j.spi.LoggingEvent;
+
+
+public class InstrumentedAppender extends AppenderSkeleton {
+    
+	private final CollectorRegistry registry;
+
+    private Counter all;
```

Comment:
The only time to do aggregation in the client is when you're running into performance issues due to high cardinality. That's not going to be a problem here, so there's no need to have an `all`.